From c9ba6e820220811009b59df40c386fcbd5efc961 Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Fri, 19 Feb 2016 13:02:36 +0800
Subject: [PATCH 61/65] package-kernel: update to v4.4-novena-r8

This version has imporived serial suspend/resume, as well as fixing a
glitch in the ldb controller.

Signed-off-by: Sean Cross <xobs@kosagi.com>
---
 package-kernel.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package-kernel.sh b/package-kernel.sh
index 43d9b51..0cba01c 100755
--- a/package-kernel.sh
+++ b/package-kernel.sh
@@ -1,7 +1,7 @@
 #!/bin/sh -e
 # A script to build the kernel .deb files, as used by Kosagi.
 threads=$(($(grep '^processor' /proc/cpuinfo  | cut -d: -f2 | tail -n1)+1))
-version=4.4-novena-r7
+version=4.4-novena-r8
 
 if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
 	echo "Repo has local changes.  Stash or commit them."
-- 
2.7.3

